lang/cs/Org.Apache.REEF.Examples/Org.Apache.REEF.Examples.csproj

```diff
@@ -94,6 +94,9 @@ under the License.
     <None Include="ConfigFiles\evaluator.conf">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="ConfigFiles\evaluatorWithService.conf">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
I removed DeploymentItem in the test code and added reference to the config files in evaluator.Tests.